contracts/strategies/RFPStrategy.sol

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.19;
+
+import {BaseStrategy} from "./BaseStrategy.sol";
+import {Metadata} from "../core/libraries/Metadata.sol";
+import {ReentrancyGuard} from "@openzeppelin/security/ReentrancyGuard.sol";
+
+contract RFPStrategy is BaseStrategy, ReentrancyGuard {
+    /// @notice Struct to hold details of an recipient
+    struct Recipient {
+        address payoutAddress;
+        uint256 proposedAmount;
+        RecipientStatus recipientStatus;
+    }
+
+    struct Payout {
+        address recipient;
+        uint256 amount;
+    }
+
+    /// ===============================
+    /// ========== Errors =============
+    /// ===============================
+
+    error READY_FOR_PAYOUT();
+    error NOT_READY_FOR_PAYOUT();
+    error NOT_IMPLEMENTED();
+    error NOT_ELIGIBLE();
+    error UNAUTHORIZED();
+
+    /// ================================
+    /// ========== Storage =============
+    /// ================================
+
+    bool public payoutReady;
+    uint256 private _counter;
+    address public acceptedRecipientId;
+
+    /// @notice recipientId -> Recipient
+    mapping(address => Recipient) public recipients;
+
+    /// ======================
+    /// ======= Events =======
+    /// ======================
+
+    event RecipientRegistered(
+        address indexed recipientId, address payoutAddress, Metadata metadata, address indexed sender
+    );
+    event Allocated(bytes data, address indexed allocator);
+    event Distributed(address indexed recipientId, address recipient, uint256 amount, address token);
+
+    /// ===============================
+    /// ======== Constructor ==========
+    /// ===============================
+
+    constructor(address _allo, string memory _name) BaseStrategy(_allo, _name) {}
+
+    /// ====================================
+    /// =========== Functions ==============
+    /// ====================================
+
+    function initialize(uint256 _poolId, bytes memory _data) public override {
+        super.initialize(_poolId, _data);
+    }
```

Comment:
I think you can skip this — if it's just calling `super.initialize()` then not including the function will do the same thing (default to the function in the inherited contract).